diff.js

```diff
@@ -15,6 +19,35 @@ module.exports = function(config, done) {
 
     var discrepancies = 0;
 
+    var fileName = config.backup.prefix + +new Date() + '.json';
```

Comment:
an S3 prefix may or may not contain `/` characters that could screw up this file name if they result in a file that would live in a non-existent folder